content/en/showcase/ampio-help/index.md

```diff
@@ -0,0 +1,77 @@
+
+Custom output formats were also used to create search indexes. The search functionality is built on the brilliant [TNTSearch](https://github.com/teamtnt/tntsearch) library. The search queries and results are handled by PHP snippets embedded into static documents handled by hugo.
+
+We even implemented a simple REST API generated by hugo! We are yet to come up with a thing that cannot be achieved with this stack, while in WordPress-based solutions we were struggling with things as simple as defining custom document ordering in one of the categories list views...
+
+When talking about hugo, we cannot forget about the speed. At the beginning we were not considering it a killer feature, but as our document base grew bigger, we appreciated it more and more. Dry-runs are not so common - most of the time we are working on one of the documents with cache already built during one of the previous hugo runs. In such a scenario, hugo rebuilds the site in about a second and we consider it a very good result.
+
+```
+                   | EN  | PL
+-------------------+-----+------
+  Pages            | 483 | 486
+  Paginator pages  |  56 |  55
+  Non-page files   | 745 | 749
+  Static files     | 917 | 917
+  Processed images | 487 | 490
+  Aliases          |  80 |  79
+  Sitemaps         |   2 |   1
+  Cleaned          |   0 |   0
+
+Total in 1096 ms
+```
+
+## Adaptation among the contributors
+
+Very quickly it became apparent that our initial concerns about the adaptation of the workflow among contributors were grossly exaggerated. Markdown as such, is fairly straightforward and did not cause any trouble to the contributors.
+
+We recommended our colleagues to use Visual Studio Code as a tool for content creation. The project’s repository tracks project-scoped configuration of the editor, which includes a set of _tasks_ allowing to run a live server from the GUI level. This is very useful for the ones who are easily frightened when faced with the mighty terminal.
+
+The basics skills of the git workflow were also easily acquired. At the end of the day, builds and deployments are fully managed by CI/CD processes, so the administration of the service drills down to reviewing and accepting merge requests in the git frontend. As a side effect, we receive full and clear history of contributions, which is well appreciated by our quality assurance auditors.
+
+We could even say that our experiment spread the love for git among non-programmers in our organisation!
+
+## Summary
+
+Hugo is the best! Definitely give it a try if you are ever faced with a challenge similar to ours. And do not give it a second thought if your service contributors are not too technically inclined - it might still turn out great!
```

Comment:
```suggestion
Hugo is the best! Definitely give it a try if you are ever faced with a challenge similar to ours. And do not give it a second thought if your service contributors are not too technically inclined---it might still turn out great!
```